tests/testthat/test-matrix.r

```diff
@@ -199,7 +199,7 @@ test_that("warning is thrown if population needed but no 'year' column present",
 
 test_that("warning is thrown if day of week is asked to be weighed but not present", {
   expect_snapshot(
-    tmp <- contact_matrix(survey = polymod3, weigh.dayofweek = TRUE),
+    tmp <- contact_matrix(survey = polymod3, weigh.dayofweek = TRUE) #nolint
```

Comment:
do we actually need the assignment here? 
```suggestion
    tmp <- contact_matrix(survey = polymod3, weigh.dayofweek = TRUE) # nolint
```

Reply:
See below discussion about expect_snapshot_warning(), which resolves this :)